src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -148,6 +151,8 @@ function MoneyRequestView({
         (field: ViolationField, data?: OnyxTypes.TransactionViolation['data']): boolean => !!canUseViolations && getViolationsForField(field, data).length > 0,
         [canUseViolations, getViolationsForField],
     );
+    const noteTypeViolations = transactionViolations?.filter((violation) => violation.type === 'notice').map((v) => ViolationsUtils.getViolationTranslation(v, translate));
+    const shouldShowNotesViolations = !isReceiptBeingScanned && canUseViolations && hasReceipt;
```

Comment:
@Krishna2323 If there is no receipt in the request, I believe we do raise note type violation. `hasNoteTypeViolation` will return true in MoneyRequestPreviewContext resulting Review Required sub-state but we don't show any receipt audit message because hasReceipt is false. This would create discrepancy.